mrp_production_request/__manifest__.py

```diff
@@ -6,7 +6,7 @@
     "summary": "Allows you to use Manufacturing Request as a previous "
                "step to Manufacturing Orders for better manufacture "
                "planification.",
-    "version": "10.0.1.0.0",
+    "version": "10.0.1.0.1",
```

Comment:
First of all I think you should bump the version to `10.0.1.1.0` not `10.0.1.0.1`. See the [contributing reference](https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#version-numbers).